pkg/apis/kubeflow.org/v2alpha1/trainingruntime_types.go

```diff
@@ -23,6 +23,8 @@ import (
 )
 
 // +kubebuilder:object:root=true
+// +kubebuilder:storageversion
```

Comment:
Do we need storageversion for CRD version migrations in the future ?

Reply:
Yes, I guess that we need this storageVersion for the situation where we switched to v2apha2 or v2beta1.